### ISSUE TITLE
false answer

### ISSUE THREAD

**Issue by k1nse-1337** [kernelwernel/VMAware#4]
tells me im on an virutal machine even tho im not?

**Comment by kernelwernel** [comment]
> tells me im on an virutal machine even tho im not?

can you recompile with `cmake -DCMAKE_BUILD_TYPE=Debug ..` instead of `cmake ..` and copy paste the output?

also can i get some basic information such as what OS you used, what techniques have been falsely detected, etc...

**Comment by k1nse-1337** [comment]
after a second run and a third one it only detected vm files.

**Comment by kernelwernel** [comment]
> after a second run and a third one it only detected vm files.

Fixed, I didn't think there would be false positives for the VM file matching technique but I've lowered the score for that function. This also includes the computer name matching function but admittedly I've fucked it up due to some conditional checks I've missed.

Hopefully it shouldn't detect it now though, there will be a false positive for the technique you've mentioned but it shouldn't be a huge problem now.
